packages/frontend/app/(dashboard)/components/coursesSection.tsx

```diff
@@ -61,10 +61,19 @@ const CoursesSection: React.FC<CoursesSectionProps> = ({ courses }) => {
               </div>
             }
           >
-            <div className="flex items-center justify-between align-middle">
+            <div className="flex flex-wrap items-center justify-between align-middle">
               {/* TODO: add the course section to the name since out of all the places to put the section, it should probably show up here. */}
```

Comment:
Do we need this comment since it's in Meta?

Reply:
the course section is not shown no. The endpoint needs to be changed to return the course section

Reply:
Ah, I misread it as course title, not section :/